packages/components/src/components/progress-bar-onboarding/progress-bar-onboarding.scss

```diff
@@ -17,4 +17,8 @@
         border-radius: 50%;
         background-color: var(--fill-normal-1);
     }
+
+    &-transition {
+        transition: all 0.25s linear;
```

Comment:
@sergei-deriv you have 2 different values for transition period(0.24 & 0.25), make it consistant